School Districts (#23)
agrc/metadata-asset-tracking-tool: metadata/boundaries/school_district_boundaries.md

# Title

Utah School District Boundaries

## ID

fb17c1bb-954a-4a3b-997a-7024cae04fbc

## Brief Summary

> Polygon dataset representing more than 40 school districts in Utah.

Reviewer: **question**: should we add what types of schools since we have a few different layers?

> Polygon dataset representing more than 40 school districts in Utah.

Reviewer 2: ```suggestion
Polygon dataset representing the more than 40 school districts in Utah.
```
"The" implies it is an exhaustive dataset.

Author: I feel adding the "the" makes it sound awkward. I changed it to "Polygon dataset representing geographic boundaries of all school districts in Utah." Thoughts?

## Summary

> This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. We maintain this layer using data sent in by School District Offices and certified through the Lieutenant Governor's Office.

Reviewer 2: ```suggestion
This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. We maintain this layer using data provided by the school district offices and certified through the Lieutenant Governor's Office.
```
I like repetition/parallelism :)

## Description

### What is the dataset?

> This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. School districts are independently operated government units managed by an elected school board. The majority of school districts in Utah are county-wide, but there are a few that are smaller than the county they reside in (e.g. Alpine, Granite, and Jordan School Districts).

Reviewer 2: ```suggestion
This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. School districts are independently-operated government units managed by a publicly-elected school board. The majority of school districts in Utah are county-wide, but a few are smaller than the county they reside in (e.g. Alpine, Granite, and Jordan School Districts).
```

### What is the purpose of the dataset?

This dataset serves as a resource for mapping the geographic extent and total enrollment of school districts in Utah. School district data are used in public planning, educational funding, taxes, and many other areas.

### What does the dataset represent?

School district boundaries are represented as polygons and include the school district name and total enrollment according to the Utah State Board of Education 2015-2016 [Fingertip Facts](https://schools.utah.gov/fingertipfacts) page.

<!--- The latest Fingertip Facts sheet is from 2019-2020. Are we planning on updating the total enrollment as the fingertip facts come out? -->

### How was the dataset created?

> We developed this dataset using coordinate geometry ([COGO](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm)) from legal descriptions provided by the Utah State Tax Commission and the Utah State Board of Education. UGRC maintains this dataset by aggregating new data received by school district offices. All updates, corrections, or clarifications to the school district boundaries represented in this dataset must first be filed with the [Lieutenant Governor's](https://ltgovernor.utah.gov/) office. Once approved, UGRC adjusts the polygon boundaries accordingly.

Reviewer 2: ```suggestion
UGRC used coordinate geometry ([COGO](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm)) from legal descriptions provided by the Utah State Tax Commission and the Utah State Board of Education to create this dataset. We maintain it by aggregating new data received from school district offices. All updates, corrections, or clarifications to the school district boundaries represented in this dataset must first be filed with the [Lieutenant Governor's](https://ltgovernor.utah.gov/) office. Once approved, UGRC adjusts the polygon boundaries accordingly.
```
